docs/data/data-grid/localization/localization.md

```diff
@@ -63,7 +63,7 @@ The same import works for `DataGridPro` as it's an extension of `DataGrid`.
 ```jsx
 import { createTheme, ThemeProvider } from '@mui/material/styles';
 import { DataGrid, bgBG } from '@mui/x-data-grid';
-import { bgBG as pickersBgBG } from '@mui/x-date-pickers';
+import { bgBG as pickersBgBG } from '@mui/x-date-pickers/locales';
```

Comment:
We have a small inconsistency with the core
Maybe we can consider fixing it in v7